activestorage/test/models/preview_test.rb

```diff
@@ -70,6 +70,50 @@ class ActiveStorage::PreviewTest < ActiveSupport::TestCase
     assert_changes -> { !!preview.image.variant(transformations)&.send(:processed?) }, to: true do
       preview.processed
     end
+
+    image = read_image( preview.image.variant(transformations))
```

Comment:
@jonathanhefner i've added a test to followup on your PR here.

also below,  i've added basic tests for all of `#download` `#key` and `#url` since they were missing and they're public API.